bundles/org.openhab.binding.daikin/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -147,6 +173,172 @@
 			</options>
 		</state>
 	</channel-type>
+	
+	<channel-type id="acunit-cmpfrequency" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
Please use units where possible.
```suggestion
		<item-type>Number:Frequency</item-type>
```

You will also need to update your code accordingly to use QuantityType for this channel instead.

Reply:
Corrected to dimensionless